Tests/DependencyInjection/DoctrineMigrationsExtensionTest.php

```diff
@@ -419,6 +420,7 @@ private function getContainer(array $config, ?array $dbalConfig = null, ?array $
 
         $container->getDefinition('doctrine.migrations.dependency_factory')->setPublic(true);
         $container->getDefinition('doctrine.migrations.configuration')->setPublic(true);
+        $container->addCompilerPass(new CacheCompatibilityPass());
```

Comment:
At first I went with a `class_exists` but then thought we might as well bump the dependency on `doctrine/doctrine-bundle`